src/symbol/placement.js

```diff
@@ -170,6 +167,11 @@ class Placement {
                     offscreen = offscreen && placedIconBoxes.offscreen;
                 }
 
+                const hasTextData = Boolean(symbolInstance.collisionArrays.textBox || symbolInstance.collisionArrays.textCircles);
+                const hasIconData = Boolean(symbolInstance.collisionArrays && symbolInstance.collisionArrays.iconBox);
```

Comment:
The `symbolInstance.collisionArrays &&` was needed to satisfy flow complaining about `symbolInstance.collisionArrays` being possible undefined. I'm not sure why flow didn't complain for the line above.